src/com/theironyard/novauc/ATM.java

```diff
@@ -0,0 +1,14 @@
+package com.theironyard.novauc;
+
+
+public class ATM {
+    public static User user = new User();
+
+
+    public static void main(String[] args) throws Exception {
+       user.login();
+
```

Comment:
Peter amazing job.  Architecture of your program well exceeds the level of your peers.  Keep up the great work